From e0dcc22981258735b767e446cf3e8f4ecf1fba06 Mon Sep 17 00:00:00 2001
From: Todd Whiteman <todd.whiteman@joyent.com>
Date: Tue, 23 Apr 2019 08:10:35 -0700
Subject: [PATCH] TRITON-1622 Instance migration tests should make use of
 multiple servers Reviewed by: Trent Mick <trentm@gmail.com> Approved by:
 Trent Mick <trentm@gmail.com>

---
 lib/backends/smartos/tasks/machine_migrate.js | 20 -------------------
 package.json                                  |  2 +-
 2 files changed, 1 insertion(+), 21 deletions(-)

diff --git a/lib/backends/smartos/tasks/machine_migrate.js b/lib/backends/smartos/tasks/machine_migrate.js
index 9eb0725..1ed40af 100644
--- a/lib/backends/smartos/tasks/machine_migrate.js
+++ b/lib/backends/smartos/tasks/machine_migrate.js
@@ -413,16 +413,6 @@ function setupFilesystem(callback) {
     // Check for override of the vm uuid. Note that performing an uuid override
     // is only supported in a non-production environment.
     if (vm_uuid !== target_vm_uuid) {
-        var fileMarkerPath = '/lib/sdc/.sdc-test-no-production-data';
-        try {
-            fs.accessSync(fileMarkerPath);
-        } catch (ex) {
-            this.fatal('Cannot perform migration uuid override - ' +
-                'as non-production marker file is missing: ' +
-                fileMarkerPath);
-            return;
-        }
-
         dataset = dataset.replace(vm_uuid, target_vm_uuid);
 
         log.warn({
@@ -490,16 +480,6 @@ function removeSyncSnapshots(callback) {
     // Check for override of the vm uuid. Note that performing an uuid override
     // is only supported in a non-production environment.
     if (vm.uuid !== payload.vm_uuid) {
-        var fileMarkerPath = '/lib/sdc/.sdc-test-no-production-data';
-        try {
-            fs.accessSync(fileMarkerPath);
-        } catch (ex) {
-            self.fatal('Cannot perform migration uuid override - ' +
-                'as non-production marker file is missing: ' +
-                fileMarkerPath);
-            return;
-        }
-
         datasets = datasets.map(function (aDataset) {
             return aDataset.replace(vm.uuid, payload.vm_uuid);
         });
diff --git a/package.json b/package.json
index 9fed610..bffe8dd 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "cn-agent",
   "description": "Triton Compute Node Agent",
-  "version": "2.12.2",
+  "version": "2.12.3",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
-- 
2.21.0

